pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -214,6 +233,32 @@ func ParseNameservers(content string) ([][]byte, error) {
 	return nameservers, nil
 }
 
+func ParseSearchDomains(content string) ([]string, error) {
+	var searchDomains []string
+
+	scanner := bufio.NewScanner(strings.NewReader(content))
+
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, domainSearchPrefix) {
+			doms := strings.Fields(strings.TrimLeft(line, domainSearchPrefix))
```

Comment:
use strings.TrimPrefix instead. TrimLeft isn't doing what you think it's doing here. 

https://stackoverflow.com/questions/29187086/why-trimleft-doesnt-work-as-expected